### DISCUSSION TITLE
Cover and Scroll latency

### DISCUSSION THREAD

**Discussion by chrislxn** [MrWillCom/hexo-theme-cupertino#67] [Q&A] [answered]
大佬您好，目前在尝试使用您的主题搭建博客，有两个问题希望您能解答：

- 文章头图如何选择和展示
- 您博客中的滚动延迟效果是如何实现的，我尝试了主题文档中的Scroll Reveal，貌似没有效果。

Best wishes!

**Comment by MrWillCom** [comment]
- 文章头图遵循的是 Markdown 文件的 Front Matter 里的 `cover_image` 和 `cover_image_alt`， 分别填图像文件路径和图像的替代文本。
- 文档中的写法应该没错，不如确认下有没有正确地把配置写在主题配置而非站点配置里？

**Comment by chrislxn** [comment]
<img width="873" alt="image" src="https://github.com/user-attachments/assets/3a838a6b-4d44-4f50-ae43-80e9cf6c211d" />
这是主题目录下的配置文件，主要想实现滚动延迟和您博客上类似ajax的效果

**Reply by chrislxn** [reply, marked as answer]
Update: 在使用了2.0beta版本之后好用了
